networkx/algorithms/centrality/tests/test_current_flow_closeness.py

```diff
@@ -38,6 +38,32 @@ def test_current_flow_closeness_centrality_not_connected(self):
         with pytest.raises(nx.NetworkXError):
             nx.current_flow_closeness_centrality(G)
 
+    def test_current_flow_closeness_centrality_fewer_than_3_vertices(self):
+        G = nx.path_graph(2)
+        with pytest.raises(nx.NetworkXError):
+            nx.current_flow_closeness_centrality(G)
+
+    def test_current_flow_closeness_centrality_not_implemented_for(self):
+        G = nx.DiGraph()
+        G.add_edge(0, 1)
+        with pytest.raises(nx.NetworkXNotImplemented):
+            nx.current_flow_closeness_centrality(G)
+
+        G = nx.MultiGraph()
+        G.add_edges_from([(0, 1), (0, 1)])
+        with pytest.raises(nx.NetworkXNotImplemented):
+            nx.current_flow_closeness_centrality(G)
+
+        G = nx.MultiDiGraph()
+        G.add_edges_from([(0, 1), (0, 1)])
+        with pytest.raises(nx.NetworkXNotImplemented):
+            nx.current_flow_closeness_centrality(G)
+
+    def test_current_flow_closeness_centrality_pointless_concept(self):
+        G = nx.Graph()
+        with pytest.raises(nx.NetworkXPointlessConcept):
```

Comment:
Currently this `NetworkXPointlessConcept` is internally raised by `is_connected` and the error message is as follows:
```
networkx.exception.NetworkXPointlessConcept: Connectivity is undefined for the null graph.
```
You can consider adding an `if nx.is_empty(G)` condition before the `is_connected` "if" condition and returning a more appropriate error message.

Reply:
I see. We could duplicate the check for whether `G` is the null graph (you mention `nx.is_empty` but in my opinion the issue only concerns the null graph) and have a more appropriate error message to indicate that centrality* is undefined for the null graph.